people/compensation.md

```diff
@@ -64,5 +63,59 @@ We have [internal salary documents for the specific positions and levels](https:
 
 ## Benefits
 
-Benefits are offered by {term}`Code for Science and Society`.
-They are offered via [TriNet](https://www.trinet.com/) as described in [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+This section describes our benefits policy across all team members at 2i2c. 2i2c team members include people who are designated as employees and as contractors, depending on their role at 2i2c.
+
+Benefits are available to employees only and are managed through {term}`Code for Science and Society`’s relationship with a service partner or PEO. For example, employees who are US based and employed through {term}`Code for Science and Society`’s relationship with [Trinet](https://www.trinet.com/), will have access to health coverage and retirement benefits through Trinet. Employees based outside the US and employed through a relationship with a PEO like Remote will have access to benefits available in their region through that PEO. More details can be found at the [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+
+Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be charged as overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.
```

Comment:
Hmm, I think this this should be a bit rephrased to match the info in https://github.com/2i2c-org/meta/issues/444#issuecomment-1466964255. After reading it, it seemed like _"expenses include healthcare, accounting services, and reimbursable equipment expenses"_ are reimbursed up to 20%.

Maybe something like the following:
```suggestion
Contractors are independent and not employed by CS&S or a PEO. They are brought on to support 2i2c with their specific expertise. They are not employees, so employees benefits are not available in this situation. 2i2c estimates that the employees specific benefits account for 20% of a monthly payment. In order to support role equity, this amount will be treated as overhead and will be added on top of the contractor monthly payment.
```

Reply:
I am not accepting the proposed changes, please check my comment on the parent issue for more details: https://github.com/2i2c-org/meta/issues/444#issuecomment-1469031926.